apex-checks/src/test/resources/checks/soqlLimit.cls

```diff
@@ -0,0 +1,12 @@
+public class SomeClass {
+	
+	public static void aMethod() {
+            string incorrect = [SELECT dato FROM table1];
```

Comment:
string incorrect = [SELECT dato FROM table1];

List<table1> incorrect = [SELECT dato FROM table1];

just to be consistent with SFDC